data/json/recipes/armor/boots.json

```diff
@@ -16,6 +16,23 @@
     ]
   },
   {
+    "result": "boots_chitin",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_FEET",
+    "skill_used": "tailor",
+    "difficulty": 6,
```

Comment:
Ok, agreed about level 4. It's not per-se harmful to require `survival` as `chitin` can occasionally show up from other sources (spider basements and possibly pawn shops)